src/contexts/Reducers/DataReducer.jsx

```diff
@@ -38,6 +46,9 @@ export const DataReducer = (state, action) => {
         leafy: false,
         marrow: false,
         root: false,
+        alltime: false,
+        summer: false,
+        winter: false,
       };
     case "LOGOUT":
       return { ...state, wishlist: [], cart: [] };
```

Comment:
While logging out you can clean local storage also.
(Don't know if you have handled this but you can keep this as a suggestion)